cli/src/commands/pack.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path')
+const { reporter, chalk, exit } = require('@dhis2/cli-helpers-engine')
+const fs = require('fs-extra')
+const makeBundle = require('../lib/makeBundle.js')
+const makePaths = require('../lib/paths.js')
+
+exports.command = 'pack [folder]'
+
+exports.describe = 'Create archive from the build.'
+
+exports.builder = yargs =>
+    yargs
+        .positional('folder', {
+            describe: 'The folder to pack relative to cwd.',
+            type: 'string',
+            defaultDescription: '.',
```

Comment:
`'.'` might not be the correct default?

Reply:
For clarity, it is simply a **description** of the internal default, it will never get set to `.`, which e.g. `default: '.'` would do.

I'll remove it. I chose `.` instead of printing the full `cwd` because it's logically equivalent even if it's not technically true. Maybe it doesn't help the user at all.